src/niscope/metadata/enums_addon.py

```diff
@@ -1014,3 +1002,8 @@
         ],
     },
 }
+
+
+enums_additional_enums = {
+    'InputImpedance': {},  # Enum metadata actually contains constants.
```

Comment:
If `InputImpedance` is not in the extracted enums.py then you don't need to remove it here.